### DISCUSSION TITLE
[EXCEPTION] I can't find your contact details (Emre Tinaztepe / Binalyze)

### DISCUSSION THREAD

**Discussion by emretinaztepe** [bcakmakoglu/vue-flow#1561] [Q&A] [answered]
Hey Burak, I am super impressed with VueFlow! Would you mind pinging me over emre@binalyze.com when you have some time to chat? Sorry for using here to send a message. I had to throw an exception after searching for your email for quite sometime :):)
- Emre

**Comment by bcakmakoglu** [comment, marked as answer]
My contact info isn't publicly available for a reason 😉 
You can always join the VueFlow Discord channel and ask questions in there, that's the best way to reach me.
